build/docker/zap-baseline.py

```diff
@@ -120,7 +135,7 @@ def main(argv):
     usage()
     sys.exit(3)
     
-  if len(config) > 0 or len(generate) > 0 or len(report) > 0:
+  if len(config_file) > 0 or len(config_url) > 0 or len(generate) > 0 or len(report) > 0:
```

Comment:
- Is the check of `config_url` really needed? (It does not seem to use the directory.)
 - The variable `report` is no longer defined, should be `report_html`/`report_xml` instead?